spec/cypress/integration/accessibility.js

```diff
@@ -4,61 +4,6 @@ describe("Accessibility", () => {
     cy.checkA11y();
   });
 
-  it("Login should be accessible", () => {
-    cy.visit("/users/sign_in");
-    cy.checkA11y();
-
-    // School not registered page
-    cy.get('[name="user[email]"]').type("doesntexist@example.com{enter}");
-    cy.get(".govuk-error-summary").should(
-      "contain",
-      "Enter the email address your school used when they created your account"
-    );
-    cy.checkA11y();
-
-    cy.visit("/users/sign_in");
-    cy.checkA11y();
-
-    cy.appFactories([["create", "user", "early_career_teacher"]])
-      .as("userData")
-      .then(([user]) => {
-        cy.get('[name="user[email]"]').type(user.email);
-      });
-
-    cy.clickCommitButton();
-
-    cy.get("@userData").then(([user]) => {
-      cy.get("h1").should("contain", `Welcome ${user.full_name}`);
-    });
-
-    cy.checkA11y();
-  });
-
-  it("CIP should be accessible", () => {
```

Comment:
Are we removing this test forever? Or is it covered in other tests?

Reply:
Is the guy who complains about people not reading readmes reviewing a pull request without reading the pull request description? 😏

Reply:
Yes. Yes he is, what a hypocrite 😂